src/Sulu/Bundle/AdminBundle/Resources/js/components/ImageSectorSelection/ImageSectorSelection.js

```diff
@@ -0,0 +1,130 @@
+// @flow
+import {action, computed, observable} from 'mobx';
+import React from 'react';
+import RectangleSelection from '../RectangleSelection';
+import RoundingNormalizer from '../RectangleSelection/dataNormalizers/RoundingNormalizer';
+import type {SelectionData} from '../RectangleSelection/types';
+import log from 'loglevel';
+import {observer} from 'mobx-react';
+import selectionStyles from './imageSectorSelection.scss';
+
+@observer
+export default class ImageSectorSelection extends React.PureComponent {
+    props: {
+        /** Determines the position at which the selection box is rendered at the beginning. */
+        initialSelection?: SelectionData,
+        minWidth?: number,
+        minHeight?: number,
+        onChange?: (s: SelectionData) => void,
+        src: string,
+    };
+
+    image: Image;
+    rounding = new RoundingNormalizer();
+
+    @observable imageLoaded = false;
+    @observable containerWidth: number;
+    @observable containerHeight: number;
+
+    naturalHorizontalToReal = (h: number) => h * this.imageResizedWidth / this.image.naturalWidth;
```

Comment:
The next six methods (`naturalHorizontalToReal`, `realHorizontalToNatural`, `naturalVerticalToReal`, `realVerticalToNatural`, `naturalDataToReal` and `realDataToNatural`) are the converter methods, which convert the real data to the natural data. These methods have been moved from an own class into this component.